pom.xml

```diff
@@ -63,6 +65,25 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<!-- Retrofit Client-->
+		<!-- https://mvnrepository.com/artifact/com.squareup.retrofit2/retrofit -->
+		<dependency>
+			<groupId>com.squareup.retrofit2</groupId>
+			<artifactId>retrofit</artifactId>
+			<version>2.9.0</version>
```

Comment:
Since its the same version "2.9.0" it might be better to create a property on this POM.xml and set that value. And here you can just use "${property.name}" instead of "2.9.0"